jbpm-workitems/jbpm-workitems-rest/src/test/java/org/jbpm/process/workitem/rest/RestWorkItemHandlerTest.java

```diff
@@ -705,7 +719,9 @@ public void testPOSTOperationWithXmlTransformation() {
         workItem.setParameter("Method",
                               "POST");
         workItem.setParameter(PARAM_CONTENT_TYPE,
-                              "Application/XML;charset=utf-8");
```

Comment:
yes its backwards compa
at

Reply:
What about leaving the current tests the way they are? Since it is a backward compatible change, it should pass. And I think the newly added tests, especially `testGetContentTypeAndCharset`, test the fix very well.